browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -489,14 +489,26 @@ L.TileSectionManager = L.Class.extend({
 			return { offset: this._offset, topLeft: docTopLeft };
 		}
 
-		const newPaneCenter = new L.Point(
-			(docTopLeft.x - splitPos.x + (paneSize.x + splitPos.x) * 0.5 / scale),
-			(docTopLeft.y - splitPos.y + (paneSize.y + splitPos.y) * 0.5 / scale));
+		let newCenter;
+
+		const newZoom = this._map.getScaleZoom(scale);
+		if (this._layer.isCalc()) {
+			const zoomScaleAbs = this._map.zoomToFactor(newZoom);
+
+			const topLeftCell = this._layer.sheetGeometry.getCellFromPos(docTopLeft, 'corepixels');
+			const newTopLeftP = this._layer.sheetGeometry.getCellRect(topLeftCell.x, topLeftCell.y, zoomScaleAbs).getTopLeft();
+			newCenter = newTopLeftP.add(paneBounds.getSize().divideBy(2)).multiplyBy(app.dpiScale);
+		} else {
+			const newPaneCenter = new L.Point(
+				(docTopLeft.x - splitPos.x + (paneSize.x + splitPos.x) * 0.5 / scale),
```

Comment:
`splitPos` exists only in Calc, so no sense to have it in this branch probably? (I understand you don't want to break anything and it's just "old behavior" ?)

Also I don't see anything wrt panes in the Calc branch. Did you test with freezed rows and columns?